saw-core/src/Verifier/SAW/FiniteValue.hs

```diff
@@ -121,11 +124,16 @@ instance Show FirstOrderValue where
       FOVIntMod _ i -> shows i
       FOVWord _ x -> shows x
       FOVVec _ vs -> showString "[" . commaSep (map shows vs) . showString "]"
-      FOVArray{}  -> shows $ firstOrderTypeOf fv
+      FOVArray _kty d vs ->
+        let vs' = map showEntry $ Map.toAscList vs
+            d' = showEntry ("<default>", d)
+        in
+        showString "[" . commaSep (vs' ++ [d']) . showString "]"
```

Comment:
If I'm reading this correctly, this will print out something like:

```
[27 : True, 42 : False, <default> : True]
```

Where `27` and `42` are indices with explicit updates, and `<default>` represents all other indices. Is that correct?